Makefile

```diff
@@ -63,3 +63,25 @@ sync_models_to_s3:
 .PHONY:sync_models_from_s3
 sync_models_from_s3:
 	aws s3 sync s3://$(PROJECT_BUCKET)/models/ models/
+
+.PHONY:sync_latest_files_to_s3
```

Comment:
what is going on here? why do we go line by line?

Reply:
I was working out a way to allow us to not have to sync the entire models/data folder for PRs (which are big!) so `sync_latest_files_to_s3` will copy the files/folders listed in `latest_files.txt`. It's line by line since I don't want to sync entire folders.
Did you try `make sync_latest_files_from_s3`, would be interested to know if it's worked correctly?

Reply:
did not try but good idea. I agree we need to improve on selectively syncing files. maybe dvc is the way but what are doing also makes sense. 👍